Mathlib/Algebra/Group/Action/TypeTags.lean

```diff
@@ -55,18 +56,3 @@ instance Multiplicative.smulCommClass [VAdd α γ] [VAdd β γ] [VAddCommClass 
   ⟨@vadd_comm α β _ _ _ _⟩
 
 end
-
-/-- The tautological additive action by `Additive (Function.End α)` on `α`. -/
-instance AddAction.functionEnd : AddAction (Additive (Function.End α)) α := inferInstance
```

Comment:
Looks like this got renamed. Since it's an instance that may be ok, or should we add a `@[deprecated] alias`?